syft/messaging/promise.py

```diff
@@ -0,0 +1,155 @@
+"""This file contains the object we use to tell another worker that they will
+receive an object with a certain ID. This object will also contain a set of
+Plan objects which will be executed when the promise is kept (assuming
+the plan has all of the inputs it requires). """
+
+import syft as sy
+
+from abc import ABC
+from syft.workers.abstract import AbstractWorker
+
+
+class Promise(ABC):
```

Comment:
what is the role of ABC?

Reply:
I think it was to indicate that this class is an abstract class but as no abstract method is declared, I don't think it's useful

Reply:
Maybe simplify and detail should be abstract, what do you think?

Reply:
Yeah sounds good!